src/_common/StreamHub/StreamHub.cs

```diff
@@ -34,6 +34,11 @@ private protected StreamHub(IStreamObservable<TIn> provider)
     /// <inheritdoc/>
     public bool IsFaulted { get; private set; }
 
+    /// <summary>
+    /// Gets the name of the stream hub.
+    /// </summary>
+    public string Name { get; private protected init; } = string.Empty;
```

Comment:
I tend to avoid defaults so that initialization errors fail fast and get fixed.
I'll defer to your judgement, but I typically prefer not having a default that would make _forgetting to add the code_ fail silently.  Applies to both stream hub and buffer list.